interop/xds/server/server.go

```diff
@@ -78,8 +93,89 @@ func (s *testServiceImpl) EmptyCall(ctx context.Context, _ *testpb.Empty) (*test
 }
 
 func (s *testServiceImpl) UnaryCall(ctx context.Context, in *testpb.SimpleRequest) (*testpb.SimpleResponse, error) {
+	response := &testpb.SimpleResponse{ServerId: s.serverID, Hostname: s.hostname}
+
+behavior:
+	for _, headerVal := range getRPCBehaviorMetadata(ctx) {
+		switch {
+		//If the value matches "sleep-<int>", the server should wait the specified number of seconds before resuming behavior matching and RPC processing.
```

Comment:
Please include a space between `//` and the first character of the comment.

Here and in every other comment in this PR, please wrap them at 80-cols max. See: https://google.github.io/styleguide/go/decisions#comment-line-length